packages/poaps/docs/PoapsClient.md

```diff
@@ -0,0 +1,183 @@
+# PoapsClient
+
+## Description
+
+`PoapsClient` is a class representing a client for interacting with POAPs (Proof of Attendance Protocol tokens).
+
+## Constructor
+
+```typescript
+constructor(
+  private compassProvider: CompassProvider,
+  private tokensApiProvider: TokensApiProvider
+)
+```
+
+- `compassProvider` (`CompassProvider`): The provider for the POAP compass API.
+- `tokensApiProvider` (`TokensApiProvider`): The provider for the Tokens API.
+
+## Methods
+
+### `fetch`
+
+```typescript
+async fetch(input: FetchPoapsInput): Promise<PaginatedResult<POAP>>
+```
+
+Fetches a list of POAP tokens based on the given input criteria.
+
+#### Parameters
+
+- `input` (`FetchPoapsInput`): Criteria for fetching POAP tokens.
+
+#### Returns
+
+A promise that resolves to a paginated list of POAP tokens.
+
+### `getSecretCode`
+
+```typescript
+private async getSecretCode(mintCode: string): Promise<string>
+```
+
+Retrieves the secret code associated with a POAP code.
+
+#### Parameters
+
+- `mintCode` (`string`): The POAP code for which to get the secret.
+
+#### Returns
+
+A promise that resolves to the associated secret code.
+
+#### Throws
+
+- `CodeAlreadyMintedError`: Thrown when the POAP code has already been minted.
+- `CodeExpiredError`: Thrown when the POAP code is expired.
+
+### `getMintCode`
+
+```typescript
+async getMintCode(mintCode: string): Promise<PoapMintStatus>
+```
+
+Retrieves mint code details for a specific QR hash.
+
+#### Parameters
+
+- `mintCode` (`string`): The QR hash for which to get the mint code.
+
+#### Returns
+
+A promise that resolves to the mint code details.
+
+### `getMintStatus`
+
+```typescript
+async getMintStatus(queueUid: string): Promise<MintingStatus>
+```
+
+Fetches the current status of a mint based on its unique ID.
+
+#### Parameters
+
+- `queueUid` (`string`): The unique ID of the mint.
+
+#### Returns
+
+A promise that resolves to the current status of the mint.
```

Comment:
Same here, this doesn't say anything about the response, it just a description. Add the "bife".